### ISSUE TITLE
Can't parse version automatically 

### ISSUE THREAD

**Issue by shmuga** [Travelport-Ukraine/uapi-json#142]
```xml
<SOAP:Envelope xmlns:SOAP="http://schemas.xmlsoap.org/soap/envelope/"><SOAP:Body><SOAP:Fault><faultcode>Server.Business</faultcode><faultstring>General air service Error.</faultstring><detail><air:AvailabilityErrorInfo xmlns:air="http://www.travelport.com/schema/air_v36_0" xmlns:common_v36_0="http://www.travelport.com/schema/common_v36_0"><common_v36_0:Code>3000</common_v36_0:Code><common_v36_0:Service>AIRSVC</common_v36_0:Service><common_v36_0:Type>Business</common_v36_0:Type><common_v36_0:Description>General air service Error.</common_v36_0:Description><common_v36_0:TransactionId>22979F7A0A0759C564EBE0905941DA4B</common_v36_0:TransactionId><common_v36_0:TraceId/><air:AirSegmentError><air:AirSegment Key="2lmtfoBAAA/B7/4irAAAAA==" Group="0" Carrier="CA" FlightNumber="910" Origin="SVO" Destination="PEK" DepartureTime="2017-04-13T18:45:00.000+03:00" ArrivalTime="2017-04-14T07:15:00.000+08:00" ClassOfService="L" ChangeOfPlane="false" OptionalServicesIndicator="false"/><air:ErrorMessage>*0 AVAIL/WL OPEN*</air:ErrorMessage></air:AirSegmentError><air:AirSegmentError><air:AirSegment Key="2lmtfoBAAA/B9/4irAAAAA==" Group="0" Carrier="CA" FlightNumber="1321" Origin="PEK" Destination="CAN" DepartureTime="2017-04-14T09:00:00.000+08:00" ArrivalTime="2017-04-14T12:15:00.000+08:00" ClassOfService="L" ChangeOfPlane="false" OptionalServicesIndicator="false"/><air:ErrorMessage>*0 AVAIL/WL OPEN*</air:ErrorMessage></air:AirSegmentError></air:AvailabilityErrorInfo></detail></SOAP:Fault></SOAP:Body></SOAP:Envelope>
```

**Comment by articice** [comment]
@shmuga the problem here was not in version auto-detection, but incorrect error handling for a <detail>  `common_v36_0:Code` enclosed into <air:AvailabilityErrorInfo>; which I fixed in #272.
Your auto-detection code works fine.

I've added two unit tests, one that demonstrates correct error handling, second - correct version auto-detection.

**Comment by articice** [comment]
Was fixed and merged as part of PR #272
